BaseProject.xcodeproj/project.pbxproj

```diff
@@ -486,10 +486,9 @@
 			baseConfigurationReference = CEB934DF1D21A47200DF75C2 /* Development.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = "AppIcon-development";
-				CURRENT_BUILD_NUMBER = 0;
```

Comment:
I'm not sure this should be erased, isn't it used in some lane to keep track of the latest build and automatically commit the new build?

Reply:
Now the build numbers are fetched from ItunesConnect using [Spaceship](https://github.com/fastlane/fastlane/tree/master/spaceship) =)

Reply:
@dtilve check line 144 of the Fastfile. Before we were getting the last build number from build settings but that's not the case anymore, as we are now using the action ```latest_testflight_build_number``` which grabs the last build number of a certain version. We don't need that build setting no more.

Reply:
That's greattt!! 👏 👍